### ISSUE TITLE
don't support channel padding

### ISSUE THREAD

**Issue by xudh1991** [alibaba/MNN#268]
pytorch
yolov3
https://github.com/eriklindernoren/PyTorch-YOLOv3

convert finish

The error occurred at benchmark and demo
Currently padding NC4HW4 don't support channel padding
The Creator Don't support type 45, 97
Currently padding NC4HW4 don't support channel padding
The Creator Don't support type 45, 97
Resize error for 97, code=2

**Comment by yizhaoyanbo** [comment]
I have the same problem

**Comment by wlguan** [comment]
Hi, Did you solve this problem? @xudh1991

**Comment by xudh1991** [comment]
No, Look at #74.There are many similar problems. I've started trying to use tensorflow and tensorRT
